cranelift/src/bugpoint.rs

```diff
@@ -839,6 +839,8 @@ fn reduce(isa: &dyn TargetIsa, mut func: Function, verbose: bool) -> Result<(Fun
     }
 
     try_resolve_aliases(&mut context, &mut func);
+    // Remove SourceLocs to make reduced clif IR easier to read
+    func.srclocs.clear();
```

Comment:
It isn't clear to me that we should remove them all at once. There might be some panic due to the way we are processing a particular series of srclocs. Can we try removing subset(s) of them incrementally? (I have never touched bugpoint fwiw, so I don't have  a sense of the difficulty of what I am asking and if it isn't practical then we can land this as-is).

Reply:
I think the approach used by `try_resolve_aliases` would work here: Try removing all srclocs and see if the bug reproduces. If not revert the change and keep them for the remainder of the run.